CMakeLists.txt

```diff
@@ -111,8 +111,9 @@ if(WIN32)
        src/win/util.c
        src/win/winapi.c
        src/win/winsock.c)
-  list(APPEND uv_test_libraries ws2_32)
-  list(APPEND uv_test_sources src/win/snprintf.c test/runner-win.c)
+  list(APPEND uv_test_sources
+    ${PROJECT_SOURCE_DIR}/src/win/snprintf.c
+    ${PROJECT_SOURCE_DIR}/test/runner-win.c)
```

Comment:
Can you line up the arguments?

Reply:
Sure thing!

Reply:
Actually on second thought, I'll place these in a generator expression inside the `test/CMakeLists.txt` file. Then we won't need this at all